app/swagger/schemas/address.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module Swagger
+  module Schemas
+    class Address
+      include Swagger::Blocks
+
+      swagger_schema :Address do
+        key :required, [:data]
+
+        property :data, type: :object do
+          key :required, [:attributes]
+          property :attributes, type: :object do
+            property :type, type: :string, enum:
```

Comment:
This looks like code duplication: 
https://github.com/department-of-veterans-affairs/vets-api/pull/1277/files#diff-a139928716bfddee50f8d31512274cf5R88

Perhaps put in it's own class/module for re-use?